index.md

```diff
@@ -17,7 +17,7 @@ Join #cri-o on IRC (freenode)
 
 ### Distribution Packaging
 
-* **Fedora**: Available on Fedora 25, 26, 27 and Rawhide
+* **Fedora**: Available on Fedora 25, 26, 27, 28, 29 and Rawhide
```

Comment:
I think we should drop the unsupported versions of Fedora, since we don't update them any more.
Just have 28 and 29.

`Available on all supported Fedora versions 28, 29 and Rawhide.`

Reply:
changed and pushed.